TypeSystem/Serialization/test.cc

```diff
@@ -60,6 +60,14 @@ CURRENT_STRUCT(Serializable) {
   bool operator<(const Serializable& rhs) const { return i < rhs.i; }
 };
 
+CURRENT_STRUCT(Int) {
+  CURRENT_FIELD(x, int32_t, 0);
```

Comment:
Good catch!

Just realized this CR has already been merged. The fix is going live with https://github.com/C5T/Current/pull/634.

Telepathy and good engineering are orthogonal. And it's never a wrong time for the right comment. Thanks!